package.json

```diff
@@ -58,6 +58,10 @@
     "eslint-plugin-react": "^7.10.0",
     "eslint-plugin-standard": "^3.1.0",
     "gh-pages": "^1.2.0",
+    "husky": "^1.3.1",
```

Comment:
We need to make sure we're adding these dependencies in the correct place. I'm guessing most of these are probably devDependencies. @MikeLockz